main.cpp

```diff
@@ -51,107 +51,115 @@ arm.flex();                             // 3)
 arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
-//  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  1) 
+//  Noun: cat
+//  action 1: the cat purrs
+    cat.purr()
+//  action 2: the cat stretches
+    cat.stretch();
+//  action 3: the cat chases a ball
+    cat.chaseBall
+
+//  2) 
+//  Noun: lightning
+//  action 1: the lightning strikes the ground
+    lightning.strikeGround();
+//  action 2: the lightning strikes a tree    
+    lightning.strikeTree();
+//  action 3: the lightning dissapates  
+    lightning.dissapate();
 
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: puck
+//  action 1: the puck bounces
+    puck.bounce();
+//  action 2: the puck slides
+    puck.slide();
+//  action 3: the puck stops
+    puck.stop();
+
+//  4) 
+//  Noun: phone
+//  action 1: the phone rings
+    phone.ring();
+//  action 2: the phone vibrates
+    phone.vibrate();
+//  action 3: the phone lights up
+    phone.illuminate();
 
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: bobblehead
+//  action 1: the bobblehead bobbles
+    bobblehead.bobble();
+//  action 2: the bobblehead wobbles
+    bobblehead.wobble();
+//  action 3: the bobblehead falls over
+    bobblehead.fallOver();
 
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: oven
+//  action 1: the oven preheats
+    oven.preheat();
+//  action 2: the oven smokes
+    oven.emitSmoke();
+//  action 3: the oven cools
+    oven.coolDown();
 
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: slot machine
+//  action 1: the slot machine reels spin
+    slotMachineReels.spin();
+//  action 2: the slot machine reels stop
+    slotMachineReels.stop();
+//  action 3: the slot machine bell rings
+    slotMachineBell.ring();
 
 //  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  9)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: rice krispies
+//  action 1: the rice krispies snap
+    riceKrispies.snap();
+//  action 2: the rice krispies crackle
+    riceKrispies.crackle();
+//  action 3: the rice krispies pop
+    riceKrispies.pop();
+
+//  9) 
+//  Noun: 3d printer
+//  action 1: the 3d printer moves along the X axis
+    3dPrinter.moveX();
+//  action 2: the 3d printer auto-levels
+    3dPrinter.autoLevel();
+//  action 3: the 3d printer shuts off
+    3dPrinter.off();

```

Comment:
Numbers can't be the first character in the name of a variable
- [ ] revise

Reply:
Fixed.